src/components/AuthProvider/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
why not use `import { node } from 'prop-types';`?

Reply:
If it was inside a `propTypes.js` then i would however inside a component its nice to keep the context of what `node` actually is as this might be a variable we'd want to use.